src/app/ApplicationTemplate.Presentation/ViewModels/DadJokes/DadJokesPageViewModel.cs

```diff
@@ -67,7 +67,7 @@ private async Task SetupFavoritesUpdate(CancellationToken ct)
 		void UpdateItemViewModels(IChangeSet<DadJokesQuote> changeSet)
 		{
 			var quotesVMs = Jokes.State.Data;
-			if (quotesVMs != null && quotesVMs.Any())
+			if (quotesVMs != null && quotesVMs.Length != 0)
```

Comment:
shouldn't this be > 0 ?

Reply:
Your question is legit. It's from this PR #406 though. @Soap-141 seems to be the author. What do you think?

Reply:
In my opinion, this should be fixed on `main` and not here.

Reply:
It already on main.

Reply:
Also see https://stackoverflow.com/questions/2989300/interview-question-any-vs-if-length-0-for-testing-if-a-collection-has-e